Backpack/Button/Classes/BPKButton.m

```diff
@@ -255,7 +276,7 @@ - (void)layoutSubviews {
     }
     CGFloat buttonTitleIconSpacing = [[self class] buttonTitleIconSpacing];
 
-    if (self.isTextAndIcon) {
+    if (self.isTextAndIcon || (self.isLoading && self.isTextOnly)) {
```

Comment:
Would this be cleaner?
```suggestion
    if (self.isTextAndIcon || self.isLoading) {
```

Reply:
No because `isTextAndIcon` is false when we use the dummy images

Reply:
👍